--- /usr/lib/python3/dist-packages/MySQLdb/cursors.py	2020-02-03 17:09:39.113858087 +1030
+++ /usr/lib/python3/dist-packages/MySQLdb/cursors.py.new	2020-02-03 17:09:39.117191431 +1030
@@ -124,7 +124,10 @@
                 for w in warnings:
                     self.messages.append((self.Warning, w))
                 for w in warnings:
-                    warn(w[-1], self.Warning, 3)
+                    if isinstance(w[-1], bytes):
+                        warn(w[-1].decode("utf-8"), self.Warning, 3)
+                    else:
+                        warn(w[-1], self.Warning, 3)
             elif self._info:
                 self.messages.append((self.Warning, self._info))
                 warn(self._info, self.Warning, 3)
